control_toolbox/include/control_toolbox/pid.hpp

```diff
@@ -515,47 +561,20 @@ class Pid
   void get_gains(
```

Comment:
we should also deprecate (we want to remove i_min/i_max)
`void get_gains(double & p, double & i, double & d, double & i_max, double & i_min);`
and change line
`[[deprecated("Use get_gains overload without bool antiwindup.")]]`
to 
`[[deprecated("Use get_gains overload with AntiWindupStrategy argument.")]]`